backend/src/api/users.ts

```diff
@@ -295,20 +359,19 @@ export const acceptTerms = wrapHandler(async (event) => {
 /**
  * @swagger
  *
- * /users:
- *  get:
+ * /users/search:
```

Comment:
Can we keep the `GET /users` endpoint for now? Don't want to remove the API endpoint until maybe later, once we're sure we want to deprecate it. (So, we'll have two endpoints: GET /users and POST / users/search)